schemas/0.0.1-preview/CreateUIDefinition.CommonControl.json

```diff
@@ -459,7 +459,13 @@
           "properties": {
             "required": {
               "type": [ "boolean", "string" ]
-            }
+            },
```

Comment:
This schema change only affects UserNameTextBox which I think supports regex and validationMessage.